services/frontend-service/src/ui/components/textfield/textfield.test.tsx

```diff
@@ -24,7 +24,7 @@ describe('Textfield', () => {
         // given
         const { container } = render(
             <MemoryRouter>
-                <Textfield floatingLabel="Floating label" />
+                <Textfield placeholder="Floating label" />
```

Comment:
on the search bar, can we make it so the textarea gets a thicker, blue border when selected?
Right now it looks like this:
![image](https://github.com/freiheit-com/kuberpult/assets/3481382/f56a8476-6b36-43b1-a7aa-a033f8af8df2)

Before this PR it looked like this:
![image](https://github.com/freiheit-com/kuberpult/assets/3481382/c670faba-d469-4714-85a2-9dd34c63b68a)

Reply:
Needed a bit of react, but now it looks like this when focused:
![image](https://github.com/freiheit-com/kuberpult/assets/123140/e3ad2d1f-94f0-4c7f-9be0-67cf40734b2a)

Reply:
please update the screenshot in the PR description as well.